.changes/config.json

```diff
@@ -240,6 +240,10 @@
         {
           "path": "${ pkg.path }/target/package/tauri-cli-${ pkgFile.version }.crate",
           "name": "${ pkg.pkg }-${ pkgFile.version }.crate"
+        },
+        {
+          "path": "${ pkg.path }/schema.json",
```

Comment:
Should we do the same for the cli.js release?

Reply:
no, cli.rs should be enough